.github/workflows/ci-test.yml

```
@@ -42,6 +42,47 @@ jobs:
       if: always()
       run: docker-compose -f dist/ci/docker-compose.yml down
 
+  smoke-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - registry.opensuse.org/opensuse/tumbleweed:latest
+          - registry.opensuse.org/opensuse/leap:15.5
+
+    container: ${{ matrix.image }}
+    steps:
+      - name: install git to checkout the repo
+        run: zypper -n in git
+
+      - uses: actions/checkout@v3
+
+      - name: fix the file permissions of the repository
+        run: chown -R $(id -un):$(id -gn) .
+
+      - name: add the openSUSE:Tools repository for Tumbleweed
+        run: zypper -n ar https://download.opensuse.org/repositories/openSUSE:/Tools/openSUSE_Tumbleweed/openSUSE:Tools.repo
+        if: ${{ contains(matrix.image, 'tumbleweed') }}
+      - name: add the openSUSE:Tools repository for Leap
+        run: zypper -n ar https://download.opensuse.org/repositories/openSUSE:/Tools/15.5/openSUSE:Tools.repo
+        if: ${{ contains(matrix.image, 'leap') }}
+
+      - name: install the dependencies of openSUSE-release-tool
+        run: |
+          zypper -n --gpg-auto-import-keys refresh
+          zypper -n source-install openSUSE-release-tools
```

```suggestion
          zypper -n source-install -d openSUSE-release-tools
```

for dependencies only.

But don't we actually need *runtime* requires here?

---

We do need them, that's why requests was missing.

---

So `zypper in openSUSE-release-tools` would be more correct, though one "generation" behind?

---

yes, that's now in the job as well